engine/table/src/main/java/io/deephaven/engine/util/PythonScope.java

```diff
@@ -170,7 +170,7 @@ default Map<String, Object> getEntriesMap() {
          */
     }
 
-    public PyDictWrapper globals();
+    public PyDictWrapper mainGlobals();
```

Comment:
javadoc.  It looks like you renamed it, but I'm still not certain what "it" is.  `globals` may have been a fine name.  It seems like these might be "query globals".

Reply:
it is the Python's main module namespace as opposed to any imported module's.